src/app/tutorials/angular2-whatsapp-ionic-cli/index.ts

```diff
@@ -15,7 +15,7 @@ export const ANGULAR2_WHATSAPP_IONIC_CLI: TutorialDefinition = {
   },
   versions: {
     // Git revision => version definition object
-    'master': V2_0_0,
+    'master': V3_0_0,
```

Comment:
Tags are really like branches, the difference is subtle. If fact you can still access the old "Socially Merge V2_0_0" version (0.0.7 on Tortilla): https://github.com/Urigo/Ionic2CLI-Meteor-WhatsApp/tree/00975428ffa239ce4229b29a60ddcb934d2735f3
You can even access every single commit @0.0.7: https://github.com/Urigo/Ionic2CLI-Meteor-WhatsApp/commit/f1ccf14

But since `angular-meteor-docs` right now doesn't allow to checkout a particular tag (only branches), we have to either implement it (shouldn't be difficult) or forget about uploading different versions of the same tutorial on the website.